sandbox-backend/src/main/java/com/exadel/sandbox/team5/service/impl/DiscountServiceImpl.java

```diff
@@ -26,6 +27,15 @@ public Discount getById(Long id) {
         return discountDAO.findAll();
     }
 
+    @Override
+    public List<Discount> getByNameOrDescriptionContaining(String searchWord) {
+        List<Discount> result = discountDAO.findByNameContaining(searchWord);
```

Comment:
at this moment we can use some like `findByNameContainingOrDescriptionContaining` but in the next PR we will use our custom SQL or criteria

Reply:
done